src/Interface/Modules/Render/ViewSceneControls.ui

```diff
@@ -6,14 +6,14 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>496</width>
-    <height>662</height>
+    <width>567</width>
+    <height>1101</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>496</width>
-    <height>662</height>
+    <height>1101</height>
```

Comment:
This minimum size is wrong. It should probably be reverted back to 662. The new tab just adds one button. If anything, the minimum width should be increased--I can smush the tabs in when I resize the config screen.